default/stringtables/en.txt

```diff
@@ -5032,7 +5032,14 @@ SITREP_SNOOZE_CLEAR_INDEFINITE
 '''  Show all SitReps indefinitely dismissed'''
```

Comment:
make word order consistent... "SitReps" to end, consistent with following strings